public/pages/Dashboard/Container/DashboardOverview.tsx

```diff
@@ -163,12 +176,10 @@ export function DashboardOverview() {
   }, []);
 
   useEffect(() => {
-    // this is needed because on Dashboard.tsx, getDetectorList API is called as 1s time, but it only gets 1 detector
-    // and such result is rendered to this component since this component also relies on result from getDetectorList API.
-    // And after this component does call getDetectorList to get all
-    // detectors, we need to refresh the page with result from 2nd call
-    // TODO: we need to implement namespacing for redux as per https://tiny.amazon.com/3eszqzpx/stacques4290
-    // Issue link: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/23
+    chrome.breadcrumbs.set([]);
+  });
+
+  useEffect(() => {
```

Comment:
why are you removing the breadcrumbs here? I thought we wanted to keep them for the dashboard and detector list page

Reply:
thanks for the catch. changed to show dashboard in breadcrumbs